src/components/itemContextMenu.js

```diff
@@ -214,7 +214,7 @@ export function getCommands(options) {
         });
     }
 
-    if (canEdit && item.MediaType === 'Video' && item.Type !== 'TvChannel' && item.Type !== 'Program'
+    if ((canEdit || user.Policy.EnableSubtitleManagement) && item.MediaType === 'Video' && item.Type !== 'TvChannel' && item.Type !== 'Program'
```

Comment:
I think we need a new function in the `itemHelper` for `canEditSubtitles` that falls back to returning `user.Policy.EnableSubtitleManagement` instead of the admin check.